src/pages/ctype.ts

```diff
@@ -14,14 +14,14 @@ import { getRequestJson } from '../utilities/getRequestJson';
 interface Input {
   cType: unknown;
   creator: DidUri;
-  extrinsicHash: HexString;
+  attestationsCreated: number;
   description: string;
   tags: string[];
 }
 
 export async function POST({ request, url }: APIContext) {
   try {
-    const { cType, creator, extrinsicHash, description, tags } =
+    const { cType, creator, attestationsCreated, description, tags } =
```

Comment:
Since the frontend doesn’t send this anymore, we don’t need to accept it

Reply:
Yes

Reply:
I causes a problem to make to really make `attestationsCreated` optional during the creation of cTypes (to really assign it a default value). 

ffc107b2026b7a3c272e60e514e4faa8bbcc49ef

Reply:
What if you just make it optional on the Input interface?

Reply:
It has to be optional on `CTypeDataInput` so that `CTypeModel.create` accepts it.

Reply:
I completely deleted the `attestationsCreated` from the `Input` interface. You should not be able to set this when creating a cType through our site or any http post request.